config/config.example.toml

```diff
@@ -184,6 +184,15 @@ trustpay.base_url = "https://test-tpgw.trustpay.eu/"
 trustpay.base_url_bank_redirects = "https://aapi.trustpay.eu/"
 zen.base_url = "https://api.zen-test.com/"
 
+[bank_config.online_banking_czech_republic]
```

Comment:
Add comments for configs(`bank_config.online_banking_czech_republic`, `[bank_config.online_banking_slovakia`,`bank_config.online_banking_poland`)